build/purger/Dockerfile

```diff
@@ -0,0 +1,10 @@
+ARG RAIDEN_VERSION
```

Comment:
I thiunk we should add `
LABEL maintainer="Raiden Network Team <contact@raiden.network>"` here

Reply:
Done